meetings/2022-09-08.md

```diff
@@ -44,51 +46,72 @@
+* SC: Could we update the requirement for storageDescription?
+* RG: If I'm accessing a resource and get describedby, then that's extra info that I can follow that.
+* RG: Suppose there isn't a describedby, and only storageDescription, I would think that would be applicable for all the other resources on the same storage. only when there is a deviation, that'd when I would expect a describedby. but when there is a describedby, I would get that. there would be an extra fetch any way.
+* SC: so with my dev hat on: if I see describedby, I would get it and if I find notifications channel info, good enough for me. I stop. if I don't see the link header or don't find the info I want, then ill look into storage description link if there is one.. and follow to its description and so on.
+* SC: That may cover majority of implementations but I don't have data to back that up.
+* RG: if described by why would there be storage description
+* AC: say generic term like `rdfs:label` and put some info there but then you could also have more specific property like `skos:prefLabel` which might have the same info. but what this means is that you don't know what clients may be interacting.. some clients may not know SKOS but then they'd use the RDFS label. But other clients may like the more precise term. that puts extra burden on server but making the interop more widely available.
+* SC: Proposal:
+
+PROPOSAL: Keep the requirement as is. We can drop that when we get some implementation feedback from clients.
+
+* AC: +1
+* RG: +1
+
+* RG: AC, you are saying that storage description would have more details data. isn't that more of a generic thing to do.
+* AC: I'm saying the semantics of storage description (relation) is more precise than describedby.
+* RG: I would expect more specific semantics for a resource.
+* SC: so instead of describedby, more like `notify:notificationResource` ?
+* AC: I think the idea is that the describedby is going to be specific to a resource, and storage description is for the entire storage. there are two possibilities. the information follows the links. the other is that thye are different. in the case where they are different, and in the case with describedby, it would have more information but that would up to the implementation in the end.
+* SC: similar to superset from earlier (storage + resource specific notification channel)
+* RG: ???
+* AC: A client may decide to fetch only one. A client may have different reasons to treat them differently. I think it is up to the client.
+* RG: But the client to make that information, ??? thought about how a 
```

Comment:
indecipherable
```suggestion
* RG: But the client to make that information, ??? thought about how a 
```

Reply:
@CxRes , pardon me for not capturing this properly. Feel free to make a suggestion if you recall the flow of the conversation - same goes for line 108. Okay to leave as is or omit line too.

Reply:
Is it OK if I very slightly embellish the conversation to create a better structure (I don't intend to change the purport of the conversation though)?